src/Markdown.MAML/Renderer/MamlRenderer.cs

```diff
@@ -75,331 +31,161 @@ public string MamlModelToString(IEnumerable<MamlCommand> mamlCommands)
         /// <returns></returns>
         public string MamlModelToString(IEnumerable<MamlCommand> mamlCommands, bool skipPreambula)
         {
-            _stringBuilder.Clear();
-            if (!skipPreambula)
-            {
-                _stringBuilder.AppendLine(XML_PREAMBULA);  
-            }
-
-            AddCommands(mamlCommands);
+            var document = new XDocument(
+                new XDeclaration("1.0", "utf-8", null),
```

Comment:
Turns out when you use a StringWriter, you are forced into UTF-16 since that's how strings are represented in memory in .Net. I've changed the code to use a stream writer on a memory stream and can confirm it is writing UTF-8 now